packages/fetchai/connections/p2p_libp2p/dht/dhtpeer/benchmarks_test.go

```diff
@@ -0,0 +1,378 @@
+/* -*- coding: utf-8 -*-
+* ------------------------------------------------------------------------------
+*
+*   Copyright 2018-2019 Fetch.AI Limited
+*
+*   Licensed under the Apache License, Version 2.0 (the "License");
+*   you may not use this file except in compliance with the License.
+*   You may obtain a copy of the License at
+*
+*       http://www.apache.org/licenses/LICENSE-2.0
+*
+*   Unless required by applicable law or agreed to in writing, software
+*   distributed under the License is distributed on an "AS IS" BASIS,
+*   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*   See the License for the specific language governing permissions and
+*   limitations under the License.
+*
+* ------------------------------------------------------------------------------
+ */
+
+package dhtpeer
+
+import (
+	"bufio"
+	"net"
+	"os"
+	"testing"
+
+	"libp2p_node/aea"
+	"libp2p_node/utils"
+
+	"github.com/rs/zerolog"
+)
+
+/* **********
+* How to run
+* ***********
+
+	$ go test -p 1 -count 20 libp2p_node/dht/dhtpeer/ -run=XXX  -bench .  -benchtime=20x
+
+*/
+
+var keysFilePath = "/path/to/file/benchmark_keys.txt"
```

Comment:
this path a bit confusing, probably I miss something?

Reply:
addressed